.gitignore

```diff
@@ -0,0 +1 @@
+**/.DS_Store
```

Comment:
?? delete

Reply:
Maybe it will be better save this git ignore line, cause when you push your changes from Mac, it also push system created .DS_Store files in every folder

Reply:
yes, you're right... its the first time i see it haha so, i research a little and its ok to add it in ignore files. Thanks !